bots/images/scripts/fedora.setup

```diff
@@ -115,13 +115,26 @@ dnf $DNF_OPTS -y upgrade
 dnf $DNF_OPTS -y install $TEST_PACKAGES $COCKPIT_DEPS $IPA_CLIENT_PACKAGES $ABRT_PACKAGES
 
 if [ -n "$AVOCADO" ]; then
+
+    # enable python3 avocado support repository (avocado upstream)
+    curl -f -o /etc/yum.repos.d/avocado.repo https://avocado-project.org/data/repos/avocado-fedora.repo
+
+    # HACK: use older version missing fedora-29 repository inside https://avocado-project.org/data/repos/
+    FEDORA_VERSION=`cat /etc/redhat-release | egrep -o [0-9]+`
+    if [ "$FEDORA_VERSION" -gt 28 ]; then
+        sed -i 's/$releasever/28/' /etc/yum.repos.d/avocado.repo
+    fi
+
     dnf $DNF_OPTS -y install \
         python2-avocado \
         python2-avocado-plugins-output-html \
         fontconfig \
         npm \
         python2-selenium \
-        chromium-headless
+        chromium-headless \
+        python3-avocado \
+        python3-avocado-plugins-output-html \
+        python3-selenium
```

Comment:
Do we really need both the python 2 and  3 versions? But I'm okay with cleaning them up in a separate step if that's easier.

Reply:
Also, we can now drop selenium support from Fedora 28. Your branch also needed rebasing as we recently moved selenium testing to fedora-29.

I did all that in a force-push, and trigger an image refresh now.